vtex/loaders/user.ts

```diff
@@ -10,13 +10,14 @@ interface User {
   lastName?: string;
   profilePicture?: string;
   gender?: string;
+  document?: string;
 }
 
 async function loader(
   _props: unknown,
   req: Request,
   ctx: AppContext,
-): Promise<Person | null> {
+): Promise<Person & { document?: string } | null> {
```

Comment:
@yuriassuncx there's no such `document` field in `Person` from schema.org https://schema.org/Person, however there is a `taxID` that would match your requirements.

Also, remove any mask of the document to make sure they all have same format (without any ponctuation)

123.456.789-00 => 12345678900

Reply:
> @yuriassuncx there's no such `document` field in `Person` from schema.org https://schema.org/Person, however there is a `taxID` that would match your requirements.
> 
> Also, remove any mask of the document to make sure they all have same format (without any ponctuation)
> 
> 123.456.789-00 => 12345678900

I added the TaxID typing to the types and also added the new attribute along with a regular expression that removes special characters. Can you review?